app/Http/Controllers/ClassController.php

```diff
@@ -20,23 +20,42 @@ public function index()
         return view('classes', compact ('classes'));    
     }
 
-    /**
+
+
+    /*    #2)
      * Show the form for creating a new resource.
      */
     public function create()
     {
         return view('addclass');
     }
 
-    /**
+    
+
+     /*    #3)
      * Store a newly created resource in storage.
      */
     public function store(Request $request)
     {
     
         #dd($request);
+
+        #validation:
+$data = $request->validate([
+    'classTitle' => "required|string",
+    'capacity' => "required|digits:1",
```

Comment:
hello Rowan,,
first thing about capacity, you validate that it is one digit length

let me explain why it is unreasonable
the numbers of one digits like 0,1, 2, 3, 4, 5, 6, 7, 8 and 9
first is a class with capacity 0 is logical?
and second thing is a class of capacity 20 for example is wrong?


Always make your solution logical😊